applications/plugins/CGALPlugin/src/CGALPlugin/CylinderMesh.inl

```diff
@@ -493,132 +493,63 @@ void CylinderMesh<DataTypes>::orientate()
-            sofa::helper::gl::glVertexT(points[i]);
-        glEnd();
-
-//        //bounding box
-//        glColor3f(1.0, 0.0, 0.0);
-//		glLineWidth(10);
-//        glBegin(GL_LINE_LOOP);
-//        glVertex3f((2*a-n)*t, n*t, -m*t);
-//        glVertex3f(n*t, (2*a-n)*t, -m*t);
-//        glVertex3f(n*t, (n-2*a)*t, -m*t);
-//        glVertex3f((2*a-n)*t, -n*t, -m*t);
-//        glVertex3f((n-2*a)*t, -n*t, -m*t);
-//        glVertex3f(-n*t, (n-2*a)*t, -m*t);
-//        glVertex3f(-n*t, (2*a-n)*t, -m*t);
-//        glVertex3f((n-2*a)*t, n*t, -m*t);
-//        glEnd();
-//        glBegin(GL_LINE_LOOP);
-//        glVertex3f((2*a-n)*t, n*t, m*t);
-//        glVertex3f(n*t, (2*a-n)*t, m*t);
-//        glVertex3f(n*t, (n-2*a)*t, m*t);
-//        glVertex3f((2*a-n)*t, -n*t, m*t);
-//        glVertex3f((n-2*a)*t, -n*t, m*t);
-//        glVertex3f(-n*t, (n-2*a)*t, m*t);
-//        glVertex3f(-n*t, (2*a-n)*t, m*t);
-//        glVertex3f((n-2*a)*t, n*t, m*t);
-//        glEnd();
-//        glBegin(GL_LINES);
-//        glVertex3f((2*a-n)*t, n*t, -m*t);
-//        glVertex3f((2*a-n)*t, n*t, m*t);
-//        glVertex3f(n*t, (2*a-n)*t, -m*t);
-//        glVertex3f(n*t, (2*a-n)*t, m*t);
-//        glVertex3f(n*t, (n-2*a)*t, -m*t);
-//        glVertex3f(n*t, (n-2*a)*t, m*t);
-//        glVertex3f((2*a-n)*t, -n*t, -m*t);
-//        glVertex3f((2*a-n)*t, -n*t, m*t);
-//        glVertex3f((n-2*a)*t, -n*t, -m*t);
-//        glVertex3f((n-2*a)*t, -n*t, m*t);
-//        glVertex3f(-n*t, (n-2*a)*t, -m*t);
-//        glVertex3f(-n*t, (n-2*a)*t, m*t);
-//        glVertex3f(-n*t, (2*a-n)*t, -m*t);
-//        glVertex3f(-n*t, (2*a-n)*t, m*t);
-//        glVertex3f((n-2*a)*t, n*t, -m*t);
-//        glVertex3f((n-2*a)*t, n*t, m*t);
-//        glEnd();
-//
-//		//circle
-//		glColor3f(0.0, 0.0, 1.0);
-//		int n = 1000;
-//		float R = 0.5*m_diameter.getValue();
-//		float L = 0.5*m_length.getValue();
-//		float Pi = 3.1415926536f;
-//		glBegin(GL_LINE_LOOP);
-//		for(int i=0; i<n; ++i)
-//         glVertex3f(R*cos(2*Pi/n*i), R*sin(2*Pi/n*i), L);
-//		glEnd();
-//		glLineWidth(1);
-
-//        glBegin(GL_LINES);
-//        glVertex3f(0.0, 0.0, m*t);
-//        glVertex3f((2*a-n)*t, n*t, m*t);
-//        glVertex3f(0.0, 0.0, m*t);
-//        glVertex3f(n*t, (2*a-n)*t, m*t);
-//        glVertex3f(0.0, 0.0, m*t);
-//        glVertex3f(n*t, (n-2*a)*t, m*t);
-//        glVertex3f(0.0, 0.0, m*t);
-//        glVertex3f((2*a-n)*t, -n*t, m*t);
-//        glVertex3f(0.0, 0.0, m*t);
-//        glVertex3f((n-2*a)*t, -n*t, m*t);
-//        glVertex3f(0.0, 0.0, m*t);
-//        glVertex3f(-n*t, (n-2*a)*t, m*t);
-//        glVertex3f(0.0, 0.0, m*t);
-//        glVertex3f(-n*t, (2*a-n)*t, m*t);
-//        glVertex3f(0.0, 0.0, m*t);
-//        glVertex3f((n-2*a)*t, n*t, m*t);
-//        glEnd();
-        glPointSize(1);
-        glEnable(GL_LIGHTING);
+            points = coords[i];
+        vparams->drawTool()->drawPoints(points,8,defaulttype::Vec4f(0.0, 0.0, 1.0,1));
```

Comment:
When it is possible, maybe use directly sofa::helper::types::RGBAColor for obvious color (red, blue, green, black, etc). Easier to read immediatly